solr/solr-ref-guide/src/libs.adoc

```diff
@@ -35,6 +35,10 @@ See <<taking-solr-to-production.adoc#solr-home-directory,Solr home directory>>.
 * `<core_instance>/lib/`: In a user-managed cluster or a single-node installation, you may want to add plugins just for a specific Solr core.
 Create this adjacent to the `conf/` directory; it's not present by default.
 
+* `<solr_install>/lib/`: The `.jar` files placed here are available to all Solr cores running on the node, and to node level plugins referenced in `solr.xml` -- so basically everything.
+Contrary to `<solr_home>/lib/`, this directory is always located in the install dir, so it can be used e.g. for custom
+Dockerfile to place custom plugin jars. This directory is not present by default so create it.
```

Comment:
I think that's likely a good idea, especially with a README, since it'd be strange to see an empty folder.

Reply:
If we pre-create this then lets not overlook SOLR_HOME lib as well (not in Docker but in our distribution).

Reply:
I added the empty lib/ dir with  a README in the latest commit.

Instead of adding yet another folder underneath `<solr-git>/solr/` with just a README, I put it in `solr/packaging/static`. Not very well thought through, can change that.